Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -39,17 +39,25 @@ function Get-AzureUri
 function Get-ProxyUri
 {
     param([String] [Parameter(Mandatory=$true)] $serverUrl)
+    
+    $proxyUri = (Get-Item env:AGENT_PROXYURL).value
```

Comment:
@ericsciple VSTS_HTTP_PROXY/ VSTS_HTTP_PROXY_USERNAME /VSTS_HTTP_PROXY_PASSWORD are env that customer set if they didn't use the .proxy file to config proxy, AGENT_PROXYURL is after the agent able to config proxy, then set them for task execution.